### PR TITLE
Tuples parser

### PR DESCRIPTION
**Pull request by EmiliaPaz**
Finally I was able to implement the parser. It allows nested tuples.
Some examples:
```
(1 , 5) : Tuple Int Int
(True , 1) : Tuple Bool Int
(1 , 2) + (3 , 4) : Fails Int
5 - ((1,2) + (3,4)) : Partial Int
((1,2),3) : Tuple Tuple Int Int Int
```
Also, I implemented a couple tests.

### CODE REVIEW

File: src/parser/parsing.js [hunk 1]
```diff
@@ -163,19 +164,48 @@ class ScriptParser extends Parser {
             })
         })
 
+        // $.RULE("atomicExpression", () => {
```
Comment (alexshadley) [inline src/parser/parsing.js]: might as well remove this

File: src/Main.elm [hunk 2]
```diff
@@ -79,6 +79,13 @@ update msg model =
         Ok ts ->
           let
             env = joinTermsWithTypes ts
+            -- env =  joinTermsWithTypes [("c",Left (CTerm (CInt 1))),
```
Comment (alexshadley) [inline src/Main.elm]: debug code can be removed

**Review by alexshadley — APPROVED** [review]
Awesome work getting the parser working!  I just highlighted a couple sections of commented-out that we can remove.

**Review by brian-quiroz — APPROVED** [review]
This looks good to me! It works great

**Pull request merged**